Pspcl/Pspcl.Web/Views/Account/Login.cshtml

```diff
@@ -0,0 +1,90 @@
+﻿
+@{
+    Layout = null;
+}
+
+<!DOCTYPE html>
```

Comment:
Add a new layout 'NoSideBarLayout' and use it with login view. Also, move HTML code to layout file afterwards.